mkdocs.yml

```diff
@@ -207,7 +207,7 @@ plugins:
   - search
   - swagger-ui-tag
   - mkdocs-jupyter
-  # - social
+  - ezlinks
```

Comment:
Was this removed at some point? Does it conflict with  any of the other plugins?

Also please don't remove the commented out social plugin -- we need to bring it back at some point.

Reply:
yes it got removed as part of [this change](https://github.com/roboflow/inference/commit/f4b7eedb03797b43b73c1e9c35f3ed0203734b6b#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2L169)

Reply:
we might be able to put it back now?  I think Chris removed both of them because of issues on his local env in that commit

Reply:
It made everything work when I added it back.  If we want to go without ezlink we need to change all the links to be relative or hardcoded urls for the end route instead of the md files for absolute links.  I started doing that actually but here is a lot!  and not trivial because some of them are in generated files and hard to find

Reply:
I added social plugin back, doesn't cause any issues for me locally.  For future if we need to disable it locally we can use this mechanism instead: https://squidfunk.github.io/mkdocs-material/plugins/social/#config.enabled

Reply:
ran a build and deployed docs from this branch with social and ezlinks added back in seems to work fine: https://www.opengraph.xyz/url/https%3A%2F%2Finference.roboflow.com%2Fstart%2Fgetting-started%2F